src/Model/File.php

```diff
@@ -13,6 +13,28 @@ class File extends Model
 
     public ?string $titleField = 'meta_filename';
 
+    /** @const string All uploaded files first get this status */
+    public const STATUS_DRAFT = 'draft';
+    /** @const string Not implemented */
```

Comment:
Is more than a draft status needed? Actually the linked status should be based on FK, ie. not stored explicitly, but I am ok with `draft`/`linked` statuses...

Reply:
For now i commented out unused statuses, but please leave them be.
I'm not exactly sure why these were added historically, but maybe someone relies on them. Also at least some of them could be useful when generating thumbnails for example. We can throw them out at any time later anyway :)

About FK, you're probably right, but historically this repo was implemented like this. I guess to make it kind of independent - decouple it completely from any other part of your project implementation and DB structure.

Reply:
Removed unused statuses and implemented thumbnails etc.